api/v4/common_types.go

```diff
@@ -387,11 +387,11 @@ type EsDefaults struct {
 	//     strict: Ensure that SSL is enabled
```

Comment:
Should we call out that :
For Search Head Cluster only `Ignore` option is supported by the operator.

Reply:
Hi @kumarajeet , let's take care of that in the documentation.